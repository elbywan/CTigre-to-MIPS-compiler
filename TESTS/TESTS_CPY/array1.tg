var N := 5 in
type  int_array = array of int in

var x := 0
and tab := int_array[N] of x := x+1; x
in



  for i=0 to N-1 do
    print("tab[");printint(i);print("] => ");printint(tab[i]);print("\n")
  done
