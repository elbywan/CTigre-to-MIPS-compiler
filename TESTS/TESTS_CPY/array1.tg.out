tab[0] => 1
tab[1] => 2
tab[2] => 3
tab[3] => 4
tab[4] => 5

